tests/Fixtures/e2e/Custom_tmp_dir/infection.json

```diff
@@ -8,5 +8,5 @@
     "logs": {
         "summary": "infection-log.txt"
     },
-    "tmpDir": "/tmp/infection-cache"
+    "tmpDir": "."
```

Comment:
Have you tried running `make test` with this change?

Reply:
could you please explain why `.`?

Reply:
@borNfreee because temporary folder will be erased when you'll run infection on itself. e2e will clean up temporary folder.

Reply:
I doubt this is going to work on Docker since a current directory is non-writable there.

Reply:
@sanmai locally I'm getting only
```
Phpunit_Bat_Wrapper/


In FinderException.php line 33:
                                                                                                          
  Unable to locate a phpunit executable on local system. Ensure that phpunit is installed and available. 
```

ubuntu 18.04